whenthen-web/src/pages/LoginPage/index.tsx

```diff
@@ -12,14 +13,16 @@ const LoginPage = () => {
 
   const styles = isDesktop ? stylesDesktopDefault : stylesDesktopDefault;
   return (
-    <div>
-      <h1 className={styles.mainBlock}>
-        <Link to="/createSchedule" className={styles.titleText}>
-          When Then
-        </Link>
-      </h1>
-      <LoginComponent />
-    </div>
+    <DefaultLayout>
```

Comment:
Do not apply DefaultLayout in LoginPage

Reply:
If apply the default layout here, the sidebar will appear.
But all links in the sidebar are links for the logged-in user, so it doesn't make sense UX
To apply the default layout, need to modify the layout >> two type of layout (sidebar, non-sidebar)

Reply:
> why?
If apply the default layout here, the sidebar will appear.
But all links in the sidebar are links for the logged-in user, so it doesn't make sense UX
To apply the default layout, need to modify the layout >> two type of layout (sidebar, non-sidebar)

Reply:
Okay, from a point of the view, I got what you mean
But not using any layout component on a page also needs to be avoided, @yhdev7935 I highly recommend making a new layout that does not have a sidebar component